pkg/manager/member/tidb_member_manager.go

```diff
@@ -759,6 +764,11 @@ func getNewTiDBSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap)
 			},
 		},
 	}
+
+	if tc.Spec.TiDB.TerminationGracePeriodSeconds != nil {
+		tidbSet.Spec.Template.Spec.TerminationGracePeriodSeconds = tc.Spec.TiDB.TerminationGracePeriodSeconds
+	}
```

Comment:
didn't update other components' controller logic
implement in a separate PR?

Reply:
ok, https://github.com/pingcap/tidb-operator/issues/2847